### PR TITLE
Add dependencies for scipy

### PR DESCRIPTION
**Pull request by manojnkumar**
Found this during installs on ppc64le. Without this series of dependencies, `pip install scipy` would fail.

### CODE REVIEW

**Comment by manojnkumar** [comment]
@vasukulkarni : I posted this again with more dependencies that I found.

**Comment by manojnkumar** [comment]
@clacroix12 @vasukulkarni : Could you review this simple doc change?

File: docs/getting_started.md [hunk 1]
```diff
@@ -15,6 +15,9 @@
 5. For UI testing follow the instructions in [openshift console](https://github.com/openshift/console#dependencies) repository.
    After clone of repository and setting all dependencies please create configuration file with RUN section with `openshift_console_path`
    parameter pointing to your local clone of openshift console repo. See example in `conf/ocsci/ui-testing.yaml`.
+6. Install the dependencies for scipy
```
Comment (clacroix12) [inline docs/getting_started.md:18]: I'm on OSX so I haven't had to do this during setup. @vasukulkarni can you confirm that this is necessary for users when setting up on linux? I assume it is but I just want to verify.

Comment (vasukulkarni) [reply docs/getting_started.md:18]: successfully built on fedora 28 using 
```
$ cat /etc/redhat-release 
Fedora release 28 (Twenty Eight)

$ rpm -qa | grep 'atlas\|libffi\|openssl\|gcc'
openssl-1.1.0i-1.fc28.x86_64
libffi-3.1-16.fc28.x86_64
openssl-pkcs11-0.4.8-2.fc28.x86_64
libgcc-8.3.1-2.fc28.x86_64
openssl-libs-1.1.0i-1.fc28.x86_64
```

Comment (lsmcfadden) [reply docs/getting_started.md:18]: @vasukulkarni were you able to complete the change?  @manojnkumar did you see the question ^^

Comment (lsmcfadden) [reply docs/getting_started.md:18]: @vasukulkarni is it complete?  Should this one be closed?

**Comment by petr-balogh** [comment]
Closing due to inactivity on this PR, feel free to re-open, or send the new PR.

**Pull request closed without merging**